src/metabase/query_processor.clj

```diff
@@ -15,6 +15,7 @@
              [add-settings :as add-settings]
              [annotate-and-sort :as annotate-and-sort]
              [binning :as binning]
+             [bind-effective-timezone :as bind-timezone]
```

Comment:
can we comment what the effective timezone entails? Is it the report timezone, the data timezone, the timezone my grandmother is currently in (helpfully fetched from `whereisgrandma.com/timezone`) so I can be sure not to call her too late?

Reply:
That might not be the right term for this, but I was trying to come up with a way to refer to to the function/macro that setup "all the timezones". It binds the `report-timezone` if available, or JVM timezone if not. It also binds the data timezone. I can put comments around it, but if there's a better or more descriptive name I'd be :+1: on that too.